src/main/resources/META-INF/plugin.xml

```diff
@@ -1,7 +1,7 @@
-<idea-plugin>
+<idea-plugin url="https://github.com/apache/openwhisk-intellij-plugin">
     <id>org.apache.openwhisk.intellij</id>
-    <name>DevTools for Apache Openwhisk</name>
-    <vendor email="dev@openwhisk.apache.org" url="https://github.com/apache">Apache</vendor>
+    <name>Apache OpenWhisk</name>
```

Comment:
Update it according to the one of Apache Camel.
https://github.com/camel-tooling/camel-idea-plugin/blob/master/camel-idea-plugin/src/main/resources/META-INF/plugin.xml